File: find_shirdi.py
import re

file = r'd:\HexaVisionTech\logaa holiday\src\pages\PackageDetails.tsx'
content = open(file, encoding='utf-8').read()
matches = re.finditer(r'\'(\d+)\':\s*\{\s*"title":\s*"([^"]+)"', content)
for m in matches:
    if 'Shirdi Flight Package' in m.group(2) or 'Shirdi' in m.group(2):
        print(f'{m.group(1)}: {m.group(2)}')

File: find_shirdi.js
const fs = require('fs');

const path = './src/pages/PackageDetails.tsx';
const content = fs.readFileSync(path, 'utf8');

// Find all occurrences of package blocks
let match;
const regex = /'(\d+)':\s*\{\s*"title":\s*"(.*?Shirdi.*?)"/gi;
const packages = [];
while ((match = regex.exec(content)) !== null) {
    packages.push({ id: match[1], title: match[2] });
}

console.log(JSON.stringify(packages, null, 2));
